JWBestPracticeApps/Podfile

```diff
@@ -11,7 +11,6 @@ def common_Google_IMA
 end
 
 def common_JWPlayer
-    pod 'JWPlayerKit', '~> 4.0.1'
```

Comment:
We should leave this, but possible just as `pod 'JWPlayerKit'`, so we don't have to have an update PR on all releases.

Reply:
Whoops, yeah. And I believe it should be 4.3 actually, as Cocoapods will automatically updated it between 4.3.X->4.9.X, whereas 'JWPlayerKit' would update to any version

Reply:
Sounds good 👍🏻